docs/secpoll.zone

```diff
@@ -177,11 +177,11 @@ recursor-4.0.5.security-status                          60 IN TXT "3 Upgrade now
 recursor-4.0.6.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-03.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-05.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-06.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-07.html and https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-08.html"
 recursor-4.0.7.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2017-08.html"
 recursor-4.0.8.security-status                          60 IN TXT "3 Upgrade now, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-04.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-06.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-07.html"
-recursor-4.0.9.security-status                          60 IN TXT "1 OK"
-recursor-4.1.0-alpha1.security-status                   60 IN TXT "3 Unsupported pre-release (final release is out)"
-recursor-4.1.0-rc1.security-status                      60 IN TXT "3 Unsupported pre-release (final release is out)"
-recursor-4.1.0-rc2.security-status                      60 IN TXT "3 Unsupported pre-release (final release is out)"
-recursor-4.1.0-rc3.security-status                      60 IN TXT "3 Unsupported pre-release (final release is out)"
+recursor-4.0.9.security-status                          60 IN TXT "2 Unsupported release (EOL)"
+recursor-4.1.0-alpha1.security-status                   60 IN TXT "3 Unsupported pre-release, see https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-04.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-06.html https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-07.html"
```

Comment:
Wouldn't https://doc.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-01.html also apply here?

Reply:
Ah yes, I copied form the wrong entry...